docs/operations-guide/scaling-metabase.md

```diff
@@ -0,0 +1,142 @@
+
+But first, let's make sure our Metabase application is well-tuned to scale.
+
+## Vertical scaling
+
+Vertical scaling is the brute force approach. Give Metabase more cores and memory, and it will have more resources available to do its work. If you are experiencing performance issues related to the application itself (i.e. unrelated to the breadth and magnitude of your databases), running Metabase on a more powerful machine can help improve performance.
+
+Metabase is already efficient out of the box. For example, for a starter Metabase instance on AWS, we recommend running Metabase using Elastic Beanstalk on a `t2.small` instance, and scaling up from there. That's a single core machine with 2 gigabytes of RAM. Machines with with 4-8 gigs of RAM should handle hundreds of users, and you can bump the number of cores and gigabytes of memory if needed.
+
+## Horizontal scaling
+
+Horizontal scaling involves running multiple instances of Metabase and using a load balancer to direct traffic to the instances. The primary use cases for horizontal scaling is to improve reliability (a.k.a. "high availability").
+
+Metabase is set up for horizontal scaling out of the box, so you don't need any special configuration to run multiple instances of Metabase. The user session data is stored in the external application database, so users don't have to worry about losing saved work if one machine goes down, and administrators don't have to deal with configuring sticky sessions to make sure users are connected to the right Metabase instance. The load balancer will route users to an available instance so users can keep right on working.
+
+When scaling horizontally, however, you must use an external relational database like PostgreSQL to store your application data (all of your questions, dashboards, logs, and other Metabase data), so that all instances of Metabase can share a common database. We recommend an eternal database even if you only every run one instance of Metabase, so an external database is not an added cost for horizontal scaling. 
+
+Though you can maintain your own application database, we recommend using a managed database solution with high availability, such as [AWS's Relational Database Service (RDS)](https://aws.amazon.com/rds/), [Google Cloud Platform's Cloud SQL](https://cloud.google.com/sql), [Microsoft Azure's SQL Database](https://azure.microsoft.com/en-us/services/sql-database/), or similar offering, so your application database will always be available for your Metabase instances. Managed database solutions are especially useful for Enterprise customers who take advantage of Metabase's [auditing functionality](../enterprise-guide/audit.md).
```

Comment:
Not actually sure we're compatible with Azure.

Reply:
I don't even think GCP works unless jumping through hoops.